src/hooks/useImportingWallet.ts

```diff
@@ -321,6 +323,10 @@ export default function useImportingWallet({ showImportModal = true } = {}) {
                     handleSetImporting(false);
                   }
 
+                  setTimeout(() => {
+                    handleReviewPromptAction(ReviewPromptAction.WatchWallet);
+                  }, 500);
```

Comment:
I feel like a lil longer here, id also wrap the timeout n a lil ` InteractionManager.runAfterInteractions` which will wait until nav events are done